charts/radar-self-enrolment-ui/values.yaml

```diff
@@ -227,13 +237,13 @@ networkpolicy:
 kratosAdminUrl: "http://kratos-admin:80/admin"
 
 # -- Set this to ORY Kratos's public URL
-kratosPublicUrl: "http://kratos-public:80"
+kratosPublicUrl: "https://localhost/kratos"
 
 # -- Set this to ORY Kratos's public URL accessible from the outside world.
 kratosBrowserUrl: "https://localhost/kratos"
 
 # -- Set this to ORY Hydra's Admin URL
-hydraAdminUrl: "http://hydra-admin:4445/admin"
+hydraAdminUrl: "kratos-admin"
```

Comment:
Could you double-check the variable here? Shouldn't be the same as HYDRA_ADMIN_URL env var from line 141?

Reply:
You are right perhaps, but I copied these values from the helmfile repo (so injected values). I do not think that this matters much at this point. Nevertheless, I updated the value to match the value of `HYDRA_ADMIN_URL`.

Reply:
ahh ok, I didn't realize it was copied...